tst/regression/gold_standard/make_tarball.sh

```diff
@@ -0,0 +1,26 @@
+# ========================================================================================
+# (C) (or copyright) 2021. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+  echo "Usage: ${0}: VERSION"
+  exit 1
+fi
+
+outname="parthenon_regression_gold_v$1.tgz"
+
```

Comment:
Maybe add a warning/info message that this script should be called from a "clean(ed)" directory?
E.g., if we remove a test than "old" gold files may still linger around when trying to create an archive for the new ones.

Reply:
And while we are at it: Maybe also add a comment to ensure that the content of the readme and `current_version` are updated.
Or extend the script to take care of that so that it'd take one additional string argument and update both files before creating the archive.

Reply:
I've improved the script to check that the given version matches the one in `current_version` and that an entry in `README.md` exists. The script now also outputs the list of files that go into the tarball, so one would see if files that should go in are included.